src/M6Web/Bundle/LogBridgeBundle/Formatter/ExceptionFormatter.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace M6Web\Bundle\LogBridgeBundle\Formatter;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * ExceptionFormatter
+ */
+class ExceptionFormatter extends DefaultFormatter implements ExceptionFormatterInterface
+{
+    /**
+     * @var string
+     */
+    protected $requestExceptionAttribute;
+
+    /**
+     * @param string $requestExceptionAttribute
+     */
+    public function setRequestExceptionAttribute($requestExceptionAttribute)
+    {
+        $this->requestExceptionAttribute = $requestExceptionAttribute;
+    }
+
+    /**
+     * Overload getLogContent to add details about the exception.
+     *
+     * @param Request  $request  Request service
+     * @param Response $response Response service
+     * @param array    $options  Request options
+     *
+     * @return string
+     */
+    public function getLogContent(Request $request, Response $response, array $options)
+    {
+        if ($request->attributes->has($this->requestExceptionAttribute)) {
+            $exception = $request->attributes->get($this->requestExceptionAttribute);
+
+            return parent::getLogContent($request, $response, $options) . self::getExceptionTrace($exception);
+        }
+
+        return parent::getLogContent($request, $response, $options);
```

Comment:
I dont like the double return with same call in 2 cases.

I would like to see somthing like this : 
```php
$logContent = parent::getLogContent($request, $response, $options);

if ($request->attributes->has($this->requestExceptionAttribute)) {
    $exception = $request->attributes->get($this->requestExceptionAttribute);
 
    $logContent .= self::getExceptionTrace($exception);
}

return $logContent;
```